homeassistant/components/reolink/update.py

```diff
@@ -214,15 +253,31 @@ def latest_version(self) -> str | None:
 
         return new_firmware.version_string
 
+    @property
+    def in_progress(self) -> bool | int | None:
```

Comment:
Note that we've changed the return value of this property to only allow a boolean or `None`. The update percentage should now be returned from `update_percentage`.

https://developers.home-assistant.io/blog/2024/10/31/update-entity-changes

Reply:
Thanks, I have adjusted accordingly